autobisect/bisect.py

```diff
@@ -347,20 +343,26 @@ def verify_bounds(self):
         :return: Boolean
         """
         LOG.info("Attempting to verify boundaries...")
-        status = self.test_build(self.start)
-        if status == self.BUILD_FAILED:
+        start_result = self.test_build(self.start)
+        if start_result not in (k for k in EvaluatorResult):
```

Comment:
I'd suggest just creating a sequence type from the `EvaluatorResult` enum is cleaner.

`if start_result not in set(EvaluatorResult):`

btw the cause of this is mentioned here: https://docs.python.org/3/whatsnew/3.7.html?highlight=enum#enum
Funny that it is a 3.8 change only mentioned in the 3.7 release notes.
btw the cause of this is mentioned here: https://docs.python.org/3/whatsnew/3.7.html?highlight=enum#enum
Funny that it is a 3.8 change only mentioned in the 3.7 release notes.